docs/tutorial/file.rst

```diff
@@ -147,3 +147,31 @@ Once you access the raw data, those are regular arrays which use regular array i
 
     channel_slice = file.force1x['1.5s':'20s']  # timestamps
     data_slice = file.force1x.data[20:40]  # indices into the array
+
+Calibration for force slices can be found by checking the calibration member::
+
+    file.force1x.calibration
+
+
+CorrelatedStacks
+----------------
```

Comment:
I think that this is a big feature which deserves an entire page for itself.

Reply:
Moved to its own page.